tests/http1_tests.py

```diff
@@ -264,46 +264,52 @@ def new_server(cls, server_port, client_port, tests, handler_cls=None):
             raise RuntimeError(f"TestServer failed to start due to port {server_port} already in use issue") from exc
 
     def __init__(self, server_port, client_port, tests, handler_cls=None):
```

Comment:
This is minor but can we please move the `__init__` function as the first function in this class so I can immediately see it?